### PR TITLE
revert #73

### PR DESCRIPTION
**Pull request by elharo**
@slachiewicz Please check jenkins before merging

### CODE REVIEW

**Comment by pzygielo** [comment]
Should it be #78 instead of #73 referenced?

File: pom.xml [hunk 1]
```diff
@@ -72,6 +72,12 @@
       <artifactId>jansi</artifactId>
       <version>2.2.0</version>
       <optional>true</optional>
+    </dependency>
+      <dependency>
```
Comment (MartinKanters) [inline pom.xml:76]:
```suggestion
    <dependency>
```

**Comment by MartinKanters** [comment]
Apart from the indenting suggestion, everything looks fine.
- Jenkins: https://ci-builds.apache.org/job/Maven/job/maven-box/job/maven-shared-utils/job/rb/

**Comment by pzygielo** [comment]
@slachiewicz - with 1461e4d committed - please close this PR

**Pull request closed without merging**